KomodoPlatform/komodo-wallet-desktop
# Add/order info

Closes https://github.com/KomodoPlatform/komodo-wallet-desktop/issues/2412
cc: @TheComputerGenie

To Test:
- Create an order which will not be matched (e.g. Sell 1 DOC for 10 MARTY).
- Wait for the order to revert to maker after 60 seconds.
- Click on the order to view its details
- See text for Max volume and Min volume in order details modal

![image](https://github.com/KomodoPlatform/komodo-wallet-desktop/assets/35845239/ca9a7db7-b5c8-401c-b7d5-da0851ba835a)
